src/plugins/layout.ts

```diff
@@ -67,18 +75,22 @@ export default class Layout implements IPlugin {
   private discoverDom() {
     let discoverTime = getTimestamp();
     traverseNodeTree(document, this.discoverNode.bind(this));
-    this.shadowDomConsistent = this.shadowDom.mirrorsRealDom();
-    assert(this.shadowDomConsistent, "discoverDom", "shadowDom inconsistent after dom discovery");
-    this.backfillLayoutsAsync(discoverTime, this.onDomDiscoverComplete.bind(this));
+    this.checkConsistency("Discover DOM");
```

Comment:
Switched to using enums and added some more structure to the mutation inconsistency event (mutation sequence # and number of mutation records)

---
In reply to: [132251947](https://github.com/Microsoft/clarity-js/pull/25#discussion_r132251947) [](ancestors = 132251947)